src/amenities/amenities_server/docker-compose.yaml

```diff
@@ -0,0 +1,7 @@
+services:
+  amenities:
+    build: ../src/amenities
```

Comment:
This path is `Concierge/src/amenities/src/amenities` so the docker-compose in the top-level folder won't be able to find the .yaml file
Since it's in `src/amenities/amenities_server`, you just need `..` instead of `../src/amenities`. In case you want to move this .yaml file out to `src/amenities` then ` . ` will do